root/utility/coordinates.js

```diff
@@ -8,7 +8,15 @@
  */
 
 export function formatCoordinates(coordinates: CoordinatesT) {
-  const {latitude, longitude} = coordinates;
+  // $FlowFixMe
+  if (coordinates === '') {
+    return '';
+  }
+  let {latitude, longitude} = coordinates;
+  if (typeof latitude != 'number') {
+    latitude = Number(latitude);
+    longitude = Number(longitude);
+  }
```

Comment:
This definitely shouldn't happen. :) So let's remove this and fix it on the Perl side rather than adding kludges. I'm guessing you need to change `TO_JSON` in lib/MusicBrainz/Server/Entity/Coordinates.pm to this, maybe

```Perl
    return {
        latitude => 0 + $self->latitude,
        longitude => 0 + $self->longitude,
    };
```

Reply:
I don't see this necessary as everything works out after editing edit.pm and create.pm of place entity